### PR TITLE
Jaeger-SPM-Config

### PR DESCRIPTION
**Pull request by devrimdemiroz**
# Changes

prometheus.server-url parameter and environment variable - METRICS_STORAGE_TYPE=prometheus added. So that Jaeger SPM window now shows spanmetrics.

It is not a crucial need as Jager would already be datasourced by Grafana. Nevertheless, it is nice to have.

## Merge Requirements

For new features contributions please make sure you have completed the following
essential items:

* [x] `CHANGELOG.md` updated to document new feature additions
* [ ] Appropriate documentation updates in the [docs](../docs/) folder

Maintainers will not merge until the above have been completed. If you're unsure
which docs need to be changed ping the
[@open-telemetry/demo-approvers](https://github.com/orgs/open-telemetry/teams/demo-approvers).

### CODE REVIEW

**Review by mviitane — COMMENTED** [review]
Looks good and seems to work fine!
image

@devrimdemiroz, can you still fix the yamllint problem (add spaces after commas)?

File: docker-compose.yml [hunk 1]
```diff
@@ -575,7 +575,7 @@ services:
   jaeger:
     image: jaegertracing/all-in-one
     container_name: jaeger
-    command: ["--memory.max-traces", "10000", "--query.base-path", "/jaeger/ui"]
+    command: ["--prometheus.server-url","http://${PROMETHEUS_ADDR}","--memory.max-traces", "10000", "--query.base-path", "/jaeger/ui"]
```
Comment (puckpuck) [inline docker-compose.yml:578]: With this expanding to several parameters, let's make it a multiline array instead:

```suggestion
    command: 
      - "--memory.max-traces"
      - "10000"
      - "--query.base-path"
      - "/jaeger/ui"
      - "--prometheus.server-url"
      - "http://${PROMETHEUS_ADDR}"
```

**Review by puckpuck — CHANGES_REQUESTED** [review]

**Review by devrimdemiroz — COMMENTED** [review]
works fine. Thanks.

**Review by puckpuck — APPROVED** [review]

**Comment by puckpuck** [comment]
Please add a changelog entry, and cleanup the failed checks.

**Review by mviitane — APPROVED** [review]

**Review by julianocosta89 — APPROVED** [review]

**Comment by puckpuck** [comment]
So to make this change work with the Helm chart will require an update to the Jaeger helm chart to accept passing in templated variables properly.

Managing dependencies is fun 🤪

**Pull request merged**